<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>constructionWidget</class>
 <widget class="QWidget" name="constructionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>331</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>构造实体</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_construct">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_cancelConstruct">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_point">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_pointArgument">
         <property name="text">
          <string>参数:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_pointX">
             <property name="text">
              <string>X:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_pointX">
             <property name="minimum">
              <double>-999.990000000000009</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_pointY">
             <property name="text">
              <string>Y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_pointY">
             <property name="minimum">
              <double>-999.990000000000009</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_pointZ">
             <property name="text">
              <string>Z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBox_pointZ">
             <property name="minimum">
              <double>-999.990000000000009</double>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLabel" name="label_constructPoint">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>构建点</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_line">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_lineArgument">
         <property name="text">
          <string>参数:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_lineStartPnt">
           <property name="text">
            <string>起点:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_lineStartPntX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineStartPntX">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_lineStartPntY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineStartPntY">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_lineStartPntZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineStartPntZ">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_lineEndPnt">
           <property name="text">
            <string>终点:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_lineEndPntX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineEndPntX">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_lineEndPntY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineEndPntY">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_lineEndPntZ">
           <property name="text">
            <string>Z:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_lineEndPntZ">
           <property name="minimum">
            <double>-999.990000000000009</double>
           </property>
           <property name="maximum">
            <double>999.990000000000009</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_constructLine">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>构建线</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_cylinder">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="7" column="0">
        <widget class="QLabel" name="label_cylinderPosZ">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderDirectionZ">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_cylinderPosY">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_cylinderPosX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_cylinderArgument">
         <property name="text">
          <string>参数:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_cylinderHeight">
         <property name="text">
          <string>高度:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_cylinderPos">
         <property name="text">
          <string>位置:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_cylinderDirection">
         <property name="text">
          <string>方向:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_cylinderRadius">
         <property name="text">
          <string>半径:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderDirectionY">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderDirectionX">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderPosZ">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderPosY">
         <property name="minimum">
          <double>-99.989999999999995</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderPosX">
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderHeight">
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="3">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_cylinderRadius">
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QLabel" name="label_constructCylinder">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>构建圆柱</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_construtBall">
         <property name="font">
          <font>
           <family>Microsoft YaHei UI</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>构建球</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_ballArgument">
         <property name="text">
          <string>参数:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_ballRadius">
         <property name="text">
          <string>半径:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ballRadius">
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_ballPos">
         <property name="text">
          <string>位置:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_ballPosX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ballPosX">
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_ballPosY">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ballPosY">
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_ballPosZ">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_ballPosZ">
         <property name="minimum">
          <double>-999.990000000000009</double>
         </property>
         <property name="maximum">
          <double>999.990000000000009</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
